src/test/java/chatbot/TaskListTest.java

```diff
@@ -0,0 +1,40 @@
+package chatbot;
+
+import chatbot.task.Todo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskListTest {
+
+    @Test
+    public void taskRep_emptyTaskList_exceptionThrown() {
```

Comment:
Perhaps the name test method should follow this format, "AAATest", where AAA represents the corresponding method name.